### DISCUSSION TITLE
Automatização do ambiente de desenvolvimento

### DISCUSSION THREAD

**Discussion by RatoX** [frontendbr/forum#441] [Discussão]
**N**egads,

Comecei a automatizar o setup do meu ambiente de trabalho [saca só](https://github.com/RatoX/workstation) tá bem no inicio ainda mas estou curtindo, o que vocês usam para automatizar o ambiente de vocês?

**Comment by kaisermann** [comment]
Eu fiz o famigerado `.dotfiles` (focado em macOS): https://github.com/kaisermann/.dotfiles

**Comment by rictorres** [comment]
também uso [dotfiles](https://github.com/rictorres/dotfiles) e pra cuidar da infra local dos meus projetos, uso o [azk.io](http://www.azk.io/)
